### PR TITLE
Canvas resize for axes

### PR DESCRIPTION
**Pull request by adamrneary**

### CODE REVIEW

File: src/coffee/models/chart.coffee [hunk 1]
```diff
@@ -12,8 +12,8 @@ class Tactile.Chart
     'bullet': Tactile.BulletRenderer
 
   # default values
-  margin: {top: 20, right: 20, bottom: 20, left: 20}
-  padding: {top: 10, right: 10, bottom: 10, left: 10}
+  defaultMargin: {top: 0, right: 10, bottom: 0, left: 10}
+  defaultPadding: {top: 10, right: 10, bottom: 10, left: 10}
```
Comment (adamrneary) [inline src/coffee/models/chart.coffee:15]: If the widget block into which we insert a chart has padding, do we need this at all?

Comment (ksarna) [inline src/coffee/models/chart.coffee:15]: Hmm, let me try that.

Comment (ksarna) [inline src/coffee/models/chart.coffee:15]: Seems we can live without the margins. But the paddings are handy as the space gained by them is used.

without margins & paddings
without margins & paddings

without margins but with paddings
without margins

Comment (adamrneary) [inline src/coffee/models/chart.coffee:15]: word up.

Adam Neary | CEO
adam.neary@profitably.com (mailto:adam.neary@profitably.com)
http://www.profitably.com (http://www.profitably.com/)

**Pull request merged**